homeassistant/components/mill/manifest.json

```diff
@@ -2,8 +2,8 @@
   "domain": "mill",
   "name": "Mill",
   "documentation": "https://www.home-assistant.io/integrations/mill",
-  "requirements": ["millheater==0.8.0"],
+  "requirements": ["millheater==0.8.0", "mill-local==0.1.0"],
   "codeowners": ["@danielhiversen"],
   "config_flow": true,
-  "iot_class": "cloud_polling"
+  "iot_class": "local_polling"
```

Comment:
It is now both cloud_polling and local_polling. 
What should be used here?

Reply:
My impression is that in general the home assistant stance is that local integrations are "better" than cloud ones -- so given that this does support local, that seems reasonable to use. It may just be worth spelling out the details in the documentation if only certain devices are supported, for example -- which I see you did.